release-notes/vJanuary.md

```diff
@@ -143,7 +143,7 @@ We also added two new actions, "Find Next Selection" (`kb(editor.action.nextSele
 
 ### Input Handling
 
-We changed input handling in the editor to allow software such as [UniKey](http://www.unikey.org) (used for Vietnamese input) or [AutoHotKey](https://www.autohotkey.com) (used for keyboard automation) to work with VS Code.
+We changed input handling in the editor to allow software such as [UniKey](https://www.unikey.org) (used for Vietnamese input) or [AutoHotKey](https://www.autohotkey.com) (used for keyboard automation) to work with VS Code.

```

Comment:
Checked for most recent three blogs and release notes. I'm calling it good. We don't need to go back in time.